### PR TITLE
Taxonomy export: updated exported file name

### LINKED ISSUE #697: Export of taxonomy - file name
The exported taxonomy CSV file should contain the survey name, taxonomy list name, and preferably the export date.
Example of problem: I exported two taxonomy lists: 'tree' and 'bamboo'. I cannot detect from CSV file name which is which..

### PR DESCRIPTION
**Pull request by SteRiccio**
Fixes #697

### CODE REVIEW

File: server/modules/taxonomy/api/taxonomyApi.js [hunk 1]
```diff
@@ -147,13 +151,18 @@ export const init = app => {
       try {
         const { surveyId, taxonomyUuid, draft } = Request.getParams(req)
 
-        Response.setContentTypeFile(res, `taxonomy_${taxonomyUuid}.csv`, null, Response.contentTypes.csv)
+        const survey = await SurveyService.fetchSurveyById(surveyId, draft)
+        const surveyName = Survey.getName(Survey.getSurveyInfo(survey))
+        const taxonomy = await TaxonomyService.fetchTaxonomyByUuid(surveyId, taxonomyUuid, draft)
+        const timestamp = DateUtils.format(Date.now(), 'yyyy-MM-dd_HH-mm-ss')
```
Comment (SteRiccio) [inline server/modules/taxonomy/api/taxonomyApi.js:157]: Should there be a function in DateUtils to format timestamps in that way so we can use it elsewhere?!

File: server/modules/taxonomy/api/taxonomyApi.js [hunk 2]
```diff
@@ -147,13 +151,18 @@ export const init = app => {
       try {
         const { surveyId, taxonomyUuid, draft } = Request.getParams(req)
 
-        Response.setContentTypeFile(res, `taxonomy_${taxonomyUuid}.csv`, null, Response.contentTypes.csv)
+        const survey = await SurveyService.fetchSurveyById(surveyId, draft)
+        const surveyName = Survey.getName(Survey.getSurveyInfo(survey))
+        const taxonomy = await TaxonomyService.fetchTaxonomyByUuid(surveyId, taxonomyUuid, draft)
```
Comment (minotogna) [inline server/modules/taxonomy/api/taxonomyApi.js:156]: survey and taxonomy can be fetched using Promise.all

File: server/modules/taxonomy/api/taxonomyApi.js (hunk 1 above)
Comment (minotogna) [reply server/modules/taxonomy/api/taxonomyApi.js:157]: Sure, it can be called ```DateUtils.nowFormatDefault()``` ?

Comment (SteRiccio) [reply server/modules/taxonomy/api/taxonomyApi.js:157]: ok

File: server/modules/taxonomy/api/taxonomyApi.js (hunk 2 above)
Comment (SteRiccio) [reply server/modules/taxonomy/api/taxonomyApi.js:156]: right

**Pull request merged**